docs/framework/migration-guide/versions-and-dependencies.md

```diff
@@ -25,8 +25,9 @@ The CLR is identified by its own version number. The .NET Framework version numb
 
 The tables that follow summarize .NET Framework version history and correlate each version with Visual Studio, Windows, and Windows Server. Visual Studio supports multi-targeting, so you're not limited to the version of .NET Framework that's listed.
 
-- The check mark icon ✔️ denotes OS versions on which .NET Framework is installed but must be enabled [in Control Panel](../install/dotnet-35-windows-10.md) (for Windows) or through the Server Manager (for Windows Server).
+- The check mark icon ✔️ denotes OS versions on which .NET Framework is installed by default.
 - The plus sign icon ➕ denotes OS versions on which .NET Framework doesn't come installed but can be installed.
+- The asterisk * denotes OS versions on which .NET Framework (whether preinstalled or not) must be enabled [in Control Panel](../install/dotnet-35-windows-10.md) or, for Windows Server, through the Server Manager.
```

Comment:
```suggestion
- The asterisk \* denotes OS versions on which .NET Framework (whether preinstalled or not) must be enabled [in Control Panel](../install/dotnet-35-windows-10.md) or, for Windows Server, through the Server Manager.
```

Reply:
This is actually displaying fine in the preview.

![image](https://user-images.githubusercontent.com/24882762/89931992-d233c280-dbc1-11ea-9bc5-403d6bf0bd10.png)

Reply:
Should it be pre-installed rather than preinstalled?
https://en.wikipedia.org/wiki/Pre-installed_software

Reply:
> This is actually displaying fine in the preview.

I feel it's safer to escape the asterisk, even if it displays correctly

Reply:
> Should it be pre-installed rather than preinstalled?
> https://en.wikipedia.org/wiki/Pre-installed_software

The Microsoft style guide says not to hyphenate prefixes unless it's necessary to disambiguate the word.